airflow/providers/databricks/operators/databricks.py

```diff
@@ -160,6 +162,142 @@ def get_link(
         return XCom.get_value(key=XCOM_RUN_PAGE_URL_KEY, ti_key=ti_key)
 
 
+class DatabricksJobsCreateOperator(BaseOperator):
```

Comment:
shouldnt this be `DatabricksCreateJobsOperator` instead, similar to `DatabricksSubmitRunOperator`?

Reply:
hmm based on the other example of `DatabricksReposUpdateOperator` it seems that it is a noun and then a verb. `Jobs` is the noun and `Create` is the verb for the `DatabricksJobsCreateOperator`. Let me know if we want it the other way around. @phanikumv let me know what works.

Reply:
We have `BigqueryInsertJobOperator` , `DatabricksSubmitRunOperator`  and I feel this class also should be corrected accordingly. In a separate PR you can correct `DatabricksReposUpdateOperator`

Reply:
resolved, renamed.